web-service/helpers.py

```diff
@@ -132,42 +130,48 @@ def SortByDistanceCmp(a, b):
 
 ################################################################################
 
-def FetchAndStoreUrl(siteInfo, url):
+def GetSiteInfoForUrl(url, rssi=None, txpower=None, force_update=False):
+    siteInfo = models.SiteInformation.get_by_id(url)
```

Comment:
Not quite.  We still cache the final page content, including title, description etc.  We just don't cache the redirection `Location` http headers.

What will happen is that next time a uribeacon returns e.g. a `goo.gl` link is that we will hit `goo.gl` again (which is by design!), and as long as it returns the same url as it did last time, we will look it up and find it in the cache.